circuitbreakerpanel/circuitbreakerpanel-java-mvn/circuitbreakerpanel-impl/src/main/java/com/lightbend/lagom/recipes/cbpanel/impl/repository/Repository.java

```diff
@@ -0,0 +1,6 @@
+package com.lightbend.lagom.recipes.cbpanel.impl.repository;
+
+public interface Repository {
+    
+    String getSomethingRandom(String userName);
+}
```

Comment:
I think `Repository` and `RepositoryImpl` can be merged into a single class. There's no need for an interface.

Also, if we move `CompletableFuture.completedFuture` from the invocation point into the implementation of `RepositoryImpl#getSomethingRandom` then we'll have a more realistic situation where an async call may fail: `CompletionStage<String> getSomethingRandom()`.